src/components/src/main/java/org/apache/jmeter/assertions/SMIMEAssertion.java

```diff
@@ -71,7 +71,7 @@
 class SMIMEAssertion {
 
     // Use the name of the test element, otherwise cannot enable/disable debug from the GUI
-    private static final Logger log = LoggerFactory.getLogger(SMIMEAssertionTestElement.class);
+    private static final Logger log = LoggerFactory.getLogger(SMIMEAssertionTestElement.class); // NOSONAR
```

Comment:
What NOSONAR mean here?
What is the warning?

We should refrain from "fixsing" Sonar issues by placing NOSONAR everywhere.

Reply:
The warning is that a different class is being used in the logger i.e. `SMIMEAssertionTestElement` instead of `SMIMEAssertion`. The comment above already explained why. Is there something better we could do?

Reply:
Frankly speaking, I did not understand the commend until you clarified.